openAPI/core-back-internal.yaml

```
@@ -118,10 +118,26 @@ paths:
           Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${IPVSharedAttributesFunction.Arn}/invocations
         passthroughBehavior: "when_no_match"
         type: "aws_proxy"
+  /journey/cri/start/{criId}:
+    post:
+      description: Called when the frontend begins the CRI journey.
+      responses:
+        200:
+          description: "Returns the id, authorizationUrl and request for a CRI."
```

Very minor thing, but could this be updated to be something like

```suggestion
          description: "Returns the id, ipvClientId and authorizationUrl for a CRI."
```